test/test_helper.rb

```diff
@@ -1,5 +1,5 @@
-# We don't need this anymore for Rake because it automatically adds lib/
-# to $LOAD_PATH before running the tests, but let's leave it in so we can run
+# We don"t need this anymore for Rake because it automatically adds lib/
+# to $LOAD_PATH before running the tests, but let"s leave it in so we can run
```

Comment:
```suggestion
# We don’t need this anymore for Rake because it automatically adds lib/
# to $LOAD_PATH before running the tests, but let’s leave it in so we can run
```